src/main/java/com/palehorsestudios/alone/player/Player.java

```diff
@@ -55,7 +55,7 @@ public Player(Set<Item> items) {
       this.shelter.setIntegrity(this.shelter.getIntegrity() + 1);
     }
     // boost shelter if Player has items to assist in shelter construction
-    if(items.contains(Item.PARACHUTE_CHORD) || items.contains(Item.AXE) || items.contains(Item.HATCHET) || items.contains(Item.SHOVEL) || items.contains(Item.KNIFE)) {
+    if(items.contains(Item.PARACHUTE_CHORD) || items.contains(Item.AXE) || items.contains(Item.HATCHET) || items.contains(Item.SHOVEL) || items.contains(Item.KNIFE) || items.contains(Item.TARP)){
```

Comment:
I didn't include the tarp in this one, because I didn't want them to get double points for it. If they have one of the other items they will get the point.

Reply:
Seems like the tarp is an intuitive item to have to improve your shelter with.

My instinct is to remove either axe or hatchet from the list of items.

Also, as I mentioned earlier, it might be good to condense the items list.

Fishing Kit (as opposed to line, hooks, lures)
Axe OR Hatchet
.357 magnum AND 12 rounds (as opposed to 2 different items. I don't think we've accounted for how many usages this has, anyway)
Bow AND 10 arrows (as above with revolver)